geocoding_android/android/src/main/java/com/baseflow/geocoding/GeocodeListenerAdapter.java

```diff
@@ -0,0 +1,14 @@
+package com.baseflow.geocoding;
+
+import android.location.Address;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.List;
+
+public interface GeocodeListenerAdapter {
+    void onGeocode(@Nullable List<Address> addresses);
+
+    void onError(@Nullable String errorMessage);
+}
```

Comment:
missing empty-line at the end of file:
```suggestion
}

```